🐛 [BUG]: Installation Error

### Current Behavior


```
$ make install
Building kubectl-multi...
go build -o bin/kubectl-multi main.go
go: downloading k8s.io/apimachinery v0.29.0
go: downloading k8s.io/client-go v0.29.0
go: downloading k8s.io/api v0.29.0
go: downloading k8s.io/cli-runtime v0.29.0
go: downloading k8s.io/kubectl v0.29.0
go: downloading k8s.io/component-base v0.29.0
go: downloading k8s.io/component-helpers v0.29.0
go: downloading github.com/daviddengcn/go-colortext v1.0.0
go: downloading github.com/fvbommel/sortorder v1.1.0
go: downloading github.com/moby/term v0.0.0-20221205130635-1aeaba878587
go: downloading github.com/distribution/reference v0.5.0
go: downloading sigs.k8s.io/kustomize/kustomize/v5 v5.0.4-0.20230601165947-6ce0bf390ce3
go: downloading sigs.k8s.io/kustomize/kyaml v0.14.3-0.20230601165947-6ce0bf390ce3
go: downloading github.com/lithammer/dedent v1.1.0
go: downloading k8s.io/metrics v0.29.0
go: downloading golang.org/x/sync v0.3.0
go: downloading sigs.k8s.io/kustomize/api v0.13.5-0.20230601165947-6ce0bf390ce3
go: downloading github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
go: downloading go.starlark.net v0.0.0-20230525235612-a134d8f9ddca
# kubectl-multi/pkg/cmd
pkg/cmd/get.go:261:6: syntax error: unexpected name handleAllGet, expected (
make: *** [build] Error 1
```

### Expected Behavior

Install the plugin: 
```
$ make install
```

Get nodes from all managed clusters:
```
$ kubectl multi get nodes
```

Get pods from all clusters in all namespaces:
```
$ kubectl multi get pods -A
```

Expected output:
```
CONTEXT  CLUSTER       NAME                    STATUS  ROLES          AGE    VERSION
its1     cluster1      cluster1-control-plane  Ready   control-plane  6d23h  v1.33.1
its1     cluster2      cluster2-control-plane  Ready   control-plane  6d23h  v1.33.1
its1     its1-cluster  kubeflex-control-plane  Ready   <none>         6d23h  v1.27.2+k3s1
```

### Steps to Reproduce

1. Clone the repo: 
```
$ git clone https://github.com/kubestellar/kubectl-plugin.git
```

2. Install the plugin:
```
$ make install
```

### Additional Context/Logs

```shell

```

--- a/pkg/cmd/get.go
+++ b/pkg/cmd/get.go
@@ -257,6 +257,8 @@ func handleJobsGet(tw *tabwriter.Writer, clusters []cluster.ClusterInfo, resourc
 			}
 		}
 	}
+	return nil
+}
 
 func handleAllGet(tw *tabwriter.Writer, clusters []cluster.ClusterInfo, resourceName, selector string, showLabels bool, outputFormat, namespace string, allNamespaces bool) error {
 	fmt.Println("==> Pods")

